.gitignore

```diff
@@ -16,4 +16,5 @@ coverage
 
 storybook-static
 
-junit.xml
\ No newline at end of file
+junit.xml
+*storybook.log
```

Comment:
what generates this file?

Reply:
this line you mean? i did a fresh init of storybook.